assets/js/aplus.js

```diff
@@ -600,6 +604,9 @@ $(function() {
             if (!csrfSafeMethod(settings.type) && !this.crossDomain) {
                 xhr.setRequestHeader("X-CSRFToken", getCookie('csrftoken'));
             }
+            if (true) {
```

Comment:
You need to check at least `!this.crossDomain`, but also cheking that the url starts with `/api/` would be nice too.